examples/pes/instanton/ch4h-cbe-rate/instanton/20_non_uniform_discret/input.xml

```diff
@@ -8,7 +8,7 @@
    </ffsocket>
    <system >
        <initialize nbeads='20'> 
-           <file mode='xyz' >  init.xyz </file>
+           <file mode='xyz' >  in.xyz </file>
```

Comment:
why this file has been modified?
I see the init.xyz so this folder needs some cleaning.